sql-plugin/src/main/spark301/scala/org/apache/spark/sql/rapids/shims/ShuffleManagerShim.scala

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.spark.sql.rapids.shims.spark301
+package org.apache.spark.sql.rapids.shims
```

Comment:
I'm surprised to see and some others that are combining shim layer files, we have separate tasks for that.  Does thi make 3.2 much easier?  Otherwise it might be nice to keep separate to keep this smaller

Reply:
The idea I thought was get all versions working and then combine based on needed changes, otherwise might have to change again.

Reply:
initially it was making this PR  easier because I was able to reuse many files in spark320. But in the end I am not sure , so will try to undo and minimize the PR